README.md

```diff
@@ -205,6 +206,35 @@ systemUnderTest.start();
 strimziKafkaContainer.getProxy().setConnectionCut(true);
 ```
 
+#### xi) Run a multi-node Kafka cluster
+
+To run a multi-node Kafka cluster, you can use the StrimziKafkaCluster class with the builder pattern.
+
+1. Default configuration (ZK-based)
+
+```java
+StrimziKafkaCluster kafkaCluster = new StrimziKafkaCluster.StrimziKafkaClusterBuilder()
+    .withNumberOfBrokers(3)
+    .build();
+
+kafkaCluster.start();
+```
+
+2. Specified Kafka version, KRaft-enabled and additional Kafka configuration
+
+```java
+StrimziKafkaCluster kafkaCluster = new StrimziKafkaCluster.StrimziKafkaClusterBuilder()
+    .withNumberOfBrokers(3)
+    .withKafkaVersion("3.8.0") // if not specified then latest Kafka version is selected
+    .withKraftEnabled()        // if not specified ZK-based is used
```

Comment:
Should this use the same naming as the StirmziKafkaContainer? `withKRaft`?

Reply:
I mean sure I can change it to be more consistent.